Marvel/Events/092 Battle of The Atom.cbl

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0"?>
+<ReadingList xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <Name>2013 - Battle of The Atom</Name>
+  <Books>
+    <Book Series="All-New X-Men" Number="1" Volume="2013" Year="2013" Format="Main Series">
+      <Id>10452916-45ec-446f-a2e3-c391cb28240b</Id>
```

Comment:
That's something we should test for. Since that ID is a comicrack specific field it very well may be localized to each persons install. If that is the case, removing them from the lists would likely be the best bet. Assuming other software out there doesn't expect them to be present.